DomainContent/Trivia/FourSquareTrivia/ACTriviaParticipantsReporter.js

```diff
@@ -0,0 +1,99 @@
+// ACTriviaParticipantsReporter.js
+
+//  Created by Mark Brosche on 10-16-2018
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+/* global EventBridge Users AccountServices Agent Avatar */
+
+(function() {
+    // Agent.isAvatar = true;
+    // Avatar.skeletonModelURL = 'http://hifi-content.s3.amazonaws.com/ozan/dev/avatars/invisible_avatar/invisible_avatar.fst';
+
+    var url="put_google_script_URL_here";
+    
+    var TRIVIA_CHANNEL = "TriviaChannel",
+        HOST_PERCENTAGE = 0.1,
+        IS_AC_SCRIPT = true;
+
+    var triviaMaster,
+        prizeMoney,
+        winnerID,
+        senderID;
+
+    if (IS_AC_SCRIPT) {
+        print("AC script running");
+        var messageHandler = Messages.messageReceived.connect(function(channel, message, senderUUID, localOnly) {
+            if (channel !== TRIVIA_CHANNEL) {
+                print("not on channel");
+                return;
+            } else {
+                message = JSON.parse(message);
+                print("message contents ",JSON.stringify(message));
```

Comment:
don't have to change, but just a note for the future, you can switch those two around and just print message since it already is a string

Reply:
Yeah, good catch! print removed though.